src/components/Item/component/Dropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useRef, useEffect } from "react";
+import arrow_down from "/arrow_down.svg";
+import mobile_sort from "/ic_sort.svg";
+import "./Dropdown.css";
+
+const options = [
+  { value: "recent", label: "최신순" },
+  { value: "favorite", label: "좋아요순" },
+];
+
+export default function Dropdown({ sort, setSort }) {
+  const [isOpen, setIsOpen] = useState(false);
+  const dropdownRef = useRef(null);
+
+  const handleOptionClick = (value) => {
+    setSort(value);
+    setIsOpen(false);
+  };
+
+  const selectedLabel = options.find((opt) => opt.value === sort)?.label;
+
+  // 외부 클릭 시 드롭다운 닫기
+  useEffect(() => {
+    const handleClickOutside = (e) => {
+      if (dropdownRef.current && !dropdownRef.current.contains(e.target)) {
+        setIsOpen(false);
+      }
+    };
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => document.removeEventListener("mousedown", handleClickOutside);
+  }, []);
+
+  return (
+    <div className="dropdown" ref={dropdownRef}>
+      <div className="dropdown-toggle" onClick={() => setIsOpen(!isOpen)}>
+        <span className="dropdown-label">{selectedLabel}</span>{" "}
+        <img src={arrow_down} className="arrow-icon" />
+        <img src={mobile_sort} className="dropdown-mobile-icon" />
+      </div>
+      {isOpen && (
+        <ul className="dropdown-menu">
+          {options.map((option) => (
+            <li
+              key={option.value}
+              className="dropdown-item"
+              onClick={() => handleOptionClick(option.value)}
+            >
+              {option.label}
+            </li>
```

Comment:
❗️ 수정요청
이런 클릭 가능한 요소의 경우 버튼을 사용해주세요. 더 적절한 태그가 있는데 이미지 태그에 onClick 을 주게되면 button에게 기대하는 동작을 제대로 수행하지 못합니다!

```suggestion
            <li
              key={option.value}
              className="dropdown-item"
            >
              <button type="button" value={option.value}  onClick={handleOptionClick}>{option.label}</button>
            </li>
```

위에처럼 button 태그로 바꾸시고 onClick 시 event 객체에서 button 태그의 value를 받아오는 방식으로 구현하시는 것을 추천드려요!